openpype/hosts/maya/api/lib.py

```diff
@@ -3928,7 +3928,7 @@ def get_capture_preset(task_name, task_type, subset, project_settings, log):
     }
 
     plugin_settings = project_settings["maya"]["publish"]["ExtractPlayblast"]
-    if plugin_settings["profiles"]:
+    if plugin_settings.get("profiles", []):
```

Comment:
Please don't make backwards compatibility because of AYON. AYON is not production ready and crashes because of inconsistency is only way how to find out we have something off...
```suggestion
    if plugin_settings["profiles"]:
```